commands/commands.go

```diff
@@ -8,56 +8,55 @@ import (
 	"runtime"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/codegangsta/cli"
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/docker/machine/libmachine/state"
+	"github.com/docker/machine/libmachine/swarm"
 	"github.com/skarademir/naturalsort"
 
-	"github.com/docker/machine/drivers"
-	_ "github.com/docker/machine/drivers/amazonec2"
-	_ "github.com/docker/machine/drivers/azure"
+	//_ "github.com/docker/machine/libmachine/drivers/amazonec2"
+	//_ "github.com/docker/machine/libmachine/drivers/azure"
 	_ "github.com/docker/machine/drivers/digitalocean"
-	_ "github.com/docker/machine/drivers/exoscale"
-	_ "github.com/docker/machine/drivers/generic"
-	_ "github.com/docker/machine/drivers/google"
-	_ "github.com/docker/machine/drivers/hyperv"
+	//_ "github.com/docker/machine/libmachine/drivers/exoscale"
+	//_ "github.com/docker/machine/libmachine/drivers/generic"
+	//_ "github.com/docker/machine/libmachine/drivers/google"
+	//_ "github.com/docker/machine/libmachine/drivers/hyperv"
 	_ "github.com/docker/machine/drivers/none"
-	_ "github.com/docker/machine/drivers/openstack"
-	_ "github.com/docker/machine/drivers/rackspace"
-	_ "github.com/docker/machine/drivers/softlayer"
+	//_ "github.com/docker/machine/libmachine/drivers/openstack"
+	//_ "github.com/docker/machine/libmachine/drivers/rackspace"
+	//_ "github.com/docker/machine/libmachine/drivers/softlayer"
 	_ "github.com/docker/machine/drivers/virtualbox"
-	_ "github.com/docker/machine/drivers/vmwarefusion"
-	_ "github.com/docker/machine/drivers/vmwarevcloudair"
-	_ "github.com/docker/machine/drivers/vmwarevsphere"
-
-	"github.com/docker/machine/libmachine"
-	"github.com/docker/machine/libmachine/auth"
-	"github.com/docker/machine/libmachine/swarm"
-	"github.com/docker/machine/log"
-	"github.com/docker/machine/utils"
+	//_ "github.com/docker/machine/libmachine/drivers/vmwarefusion"
+	//_ "github.com/docker/machine/libmachine/drivers/vmwarevcloudair"
+	//_ "github.com/docker/machine/libmachine/drivers/vmwarevsphere"
+
+	"github.com/docker/machine/commands/mcndirs"
+	"github.com/docker/machine/libmachine/cert"
+	"github.com/docker/machine/libmachine/host"
+	"github.com/docker/machine/libmachine/log"
+	"github.com/docker/machine/libmachine/persist"
 )
 
 var (
+	stateTimeoutDuration  = time.Second * 3
```

Comment:
Thanks @nathanleclaire - once this is merged I'll tackle the change I was working on. Thanks for all your work!